tests/cross_functional/conftest.py

```diff
@@ -225,9 +225,10 @@ def factory(
             logger.info("Cleaned up potential database clients to nbcore!")
 
         # Delete the existing cnpg cluster
-        OCP(kind=constants.CNPG_CLUSTER_KIND).delete(
-            resource_name=constants.NB_DB_CNPG_CLUSTER_NAME
-        )
+        OCP(
+            kind=constants.CNPG_CLUSTER_KIND,
```

Comment:
Please add a JIRA decorator to skip the Kafka test, as there is an open bug that has deferred it from the 4.20 release.